src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/ProxyStreamFactoryBuilder.java

```diff
@@ -103,20 +106,28 @@ public StreamFactoryBuilder setBufferPoolSupplier(
     }
 
     @Override
+    public StreamFactoryBuilder setCounterSupplier(
+        Function<String, LongSupplier> supplyCounter)
+    {
+        slabAquires = supplyCounter.apply("slab.aquires");
```

Comment:
Spelling: acquires (with a c). I don't think that's just British English.
Naming suggestion: use the word entry instead of slab, since slab is a very internal sort of term, so: entry.acquires, entry.releases
That way the full counter names in the shm files will appear to be http-cache.entry.acquires and http-cache.entry.releases